### ISSUE TITLE
False positive "no empty line within import group"

### ISSUE THREAD

**Issue by opichon** [import-js/eslint-plugin-import#2689]
This code (from a vue SFC):

```
<script lang="ts">
import {
  defineAsyncComponent,
  computed,
  markRaw,
  shallowRef,
  watch,
} from "vue";

const NullLayout = defineAsyncComponent({
  loader: () => import("./layouts/NullLayout.vue"),
  timeout: 3000,
});

export default {
  name: "Layout",
};
</script>
```

generates the following error:

```
8:1  error  There should be no empty line within import group  import/order
```

I can't see what the problem is. Is it a bug? Is my import order wrong?

**Comment by ljharb** [comment]
Since that's not valid JS, I assume you're using a vue parser? I would expect such a parser to provide an AST to eslint as if the JS code was in a file by itself - if not, then rules won't work the same with it. Have you tried filing a bug on your vue parser?
